elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -358,6 +373,25 @@ class PfeNavigation extends PFElement {
     this._activeNavigationItems.map(item => item.close());
     this.overlay = false;
   }
+
+  /**
+   * Set a global CSS variable reporting the height of this navigation item.
+   * Used to position sticky subnavigation items under this.
+   *
+   * The name of the global CSS variable is `--pfe-navigation--Height--actual`.
+   * If this nav has an `id` attribute, the id will be appended to the variable
+   * name to distinguish it from other pfe-navigation items on the page
+   * (unlikely, but imagine a demo page with 20 example pfe-navigation elements
+   * in different states, and one genuine pfe-navigation element at the top
+   * used for actual navigation).
+   */
+  _reportHeight() {
+    const cssVarName = `--pfe-navigation__${
```

Comment:
+1 Chris to moving the underscore into the ternary

Reply:
@castastrophe fixed this. Also moved this into a single line to be more readable.